regulations/templates/regulations/about.html

```diff
@@ -53,12 +52,12 @@ <h2>Subject Matter Expert Updates</h2>
       <div class="ds-l-container">
         <div class="ds-l-row">
           <div class="ds-l-col--6">
-            <h3>Links to Additional Resources <span class="indicator">Coming Soon</span></h3>
+            <h3>Links to Additional Resources</h3>
             <p>Resource links are curated and updated by a Medicaid subject matter expert on our team.</p>
-            <p>For example, when Medicaid.gov publishes a new State Medicaid Director Letter, we add a link in the sidebar of each relevant subpart.</p>
+            <p>For example, when Medicaid.gov publishes a new State Medicaid Director Letter in the <a href="https://www.medicaid.gov/new-notable/index.html" class="external" target="_blank">New & Notable Resource Center</a>, we add a link in the sidebar of each relevant subpart.</p>
```

Comment:
We should probably make the `&` into `&amp;` instead - sometimes special characters get rendered oddly and this will prevent that.

Reply:
Sounds good, done!